b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/editUserComponent.js

```diff
@@ -49,12 +49,22 @@ export default function EditUserComponent(prop) {
     const [ userRolesForForm, setUserRolesForForm ] = useState(null);
     const [ initUserRolesForForm, setInitUserRolesForForm ] = useState(null);
 
+    /**
+     * fetch all the roles in the identity server available for the logged in organization
```

Comment:
Shall we start the comment with an upper case letter and ends with a full-stop? Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b